packages/features/shell/Shell.tsx

```diff
@@ -944,6 +944,16 @@ function SideBar({ bannersHeight, user, isPlatformUser = false }: SideBarProps)
       },
       icon: "copy",
     },
+    {
+      name: "copy_referral_link",
+      href: "",
+      onClick: (e: { preventDefault: () => void }) => {
+        e.preventDefault();
+        navigator.clipboard.writeText(`https://refer.cal.com/${user?.username}`);
+        showToast(t("link_copied"), "success");
+      },
+      icon: "copy",
+    },
```

Comment:
we should only show this on cal.com use IS_CALCOM from CONSTANTS

Reply:
Fixed: https://github.com/calcom/cal.com/pull/16165/commits/2c3814d3804766831753754c67b522081a0bd218